README.md

```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+
+## Launch The Bot Using Docker
+Prerequisites
+
+Make sure you have already installed both Docker Engine and Docker Compose. You don’t need to install other programs like Python  as they are provided by Docker images.
```

Comment:
I would remove the "prerequisites", we can rephrase this line as follows if you like it:
```suggestion
1. Install [Docker](https://docs.docker.com/get-docker/).
```

Reply:
prerequisites have been removed and the changes have been made